### DISCUSSION TITLE
apcupsd wrong divisor ?

### DISCUSSION THREAD

**Discussion by F0x06** [netdata/netdata#18779] [Q&A]
Hi !

I activated the apcupsd plugin, however in my dashboard it seems that all values are divided by 100.0

![image](https://github.com/user-attachments/assets/0926ea0e-be40-4115-b931-1a3cd012cfa4)

wrong config, or this is a bug ? 

Thanks in advance.

**Comment by ilyam8** [comment]
Hi, @F0x06. The "UPS Load" is indeed divided by 100. Fixed in #18780.

> all values are divided by 100.0

What other values?

**Reply by F0x06** [reply]
Thanks for the quick reply and fix !, i will check other values tommorow :)

**Reply by F0x06** [reply]
Oh sorry, i made a typo, this is the only one, thanks again !
